GameWorld/HexGrid/DrawableHexSphere.cs

```diff
@@ -114,6 +122,46 @@ public void SetGroundHeight(int groundHeight)
             InitializeVertices();
         }
 
+        public double GetHeightFromCoveragePercent(int coveragePercent)
+        {
+            double resultHeight = 0.5f;
+            List<double> tiles_heights = new List<double>();
+
+            foreach (var t in _sphereGrid.Tiles)
+            {
+                tiles_heights.Add(t.Height);
+            }
+
+            tiles_heights.Sort();
+
+            if (0 > coveragePercent || 100 < coveragePercent)
+            {
+                resultHeight = 0.5f;
```

Comment:
I think that in case of coveragePercent < 0 or > 100 it should become either 0 or 100 (not 0.5. It's implicit) or throw exception.